README.md

```diff
@@ -19,6 +19,7 @@ Actual manuscript instances will clone this repository (see [`SETUP.md`](SETUP.m
 
 Manubot is a system for writing scholarly manuscripts via GitHub.
 Manubot automates citations and references, versions manuscripts using git, and enables collaborative writing via GitHub.
+An [overview manuscript](https://greenelab.github.io/meta-review/) presents the benefits of collaborative writing with Manubot and its unique features.
```

Comment:
Can we add a title to the link of `"Open collaborative writing with Manubot"`

Reply:
👍 added in 9f7c0d29995a52d4653c5f63e2708a8e3d91624a